docs/diagrams/system-landscape.puml

```diff
@@ -15,24 +16,37 @@
    }
 
    Boundary(hmpps, "His Majesty's Prison and Probation Service (HMPPS)", "Executive Agency") {
+      Boundary(aws, "Amazon Web Services", "AWS"){
+         System(route53, "Route 53", "Provides DNS services.")
+         System(apigateway, "API Gateway", "Routes traffic to internal Kubernetes Ingress URL.")
+         System(s3, "S3 Truststore", "Authorised client certificate storage.")
+         
+         Rel(route53, apigateway, "Forwards integration requests to")
+         Rel(apigateway, s3, "Checks certificates against")
+      }
+
+      Boundary(logging, "Logging & Metrics"){
+         System(kibana, "Kibana", "Performance metrics.")
+         System(insights, "Azure Insights", "Usage metrics.")
+         System(sentry, "Sentry", "Error logging.")
+      }
+
       Boundary(cloudplatform, "Cloud Platform", "Elastic Kubernetes Service"){
```

Comment:
I have a fun question, should Cloud Platform live within AWS? I'm not sure of my opinion if I'm honest!

Reply:
(+1 for yes)